### PR TITLE
DR-1110 Update CSS to use nhsuk-frontend node modules package

### PR DESCRIPTION
**Pull request by emab1**
## Description

Update the CSS to use the nhsuk-frontend package in node-modules
- per implementation guidelines, SASS has been added
- the next major version of SASS deprecates many features that will need to be addressed in the root package. Therefore, currently on running the frontend, there will be many deprecation warnings in the terminal

Global.css was removed as it removed the focus styling on buttons, which contradicts NHS guidelines.

## Type of changes

- [ ] Refactoring (non-breaking change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)

## Checklist

- [ ] I am familiar with the [contributing guidelines](../docs/CONTRIBUTING.md)
- [ ] I have followed the code style of the project
- [ ] I have added tests to cover my changes
- [ ] I have updated the documentation accordingly
- [ ] This PR is a result of pair or mob programming

---

## Sensitive Information Declaration

To ensure the utmost confidentiality and protect your and others privacy, we kindly ask you to NOT including [PII (Personal Identifiable Information) / PID (Personal Identifiable Data)](https://digital.nhs.uk/data-and-information/keeping-data-safe-and-benefitting-the-public) or any other sensitive data in this PR (Pull Request) and the codebase changes. We will remove any PR that do contain any sensitive information. We really appreciate your cooperation in this matter.

- [ ] I confirm that neither PII/PID nor sensitive data are included in this PR and the codebase changes.

### CODE REVIEW

**Review by mmg-nhse — APPROVED** [review]

File: src/frontend/index.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
     <meta name="description" content="Capacity Management" />
-    <link rel="stylesheet" href="/libraries/nhsuk.css" />
+    <link rel="stylesheet" href="/main.scss" />
```
Comment (Tomdango) [inline src/frontend/index.html:9]: You can replace this with importing the scss file directly inside of the base `index.tsx` - also helps keep track for live reloading of styles in dev mode.

File: src/frontend/vite.config.mts [hunk 2]
```diff
@@ -7,6 +7,13 @@ export default defineConfig({
         emptyOutDir: true,
     },
     plugins: [react()],
+    css: {
+        preprocessorOptions: {
+            scss: {
+                api: 'modern-compiler',
+            },
+        },
+    },
```
Comment (Tomdango) [inline src/frontend/vite.config.mts:16]: Shouldn't need this if it's included directly in the `index.tsx`

**Review by Tomdango — COMMENTED** [review]
Looks good, couple of small comments :)

**Review by Tomdango — APPROVED** [review]
LGTM 👍

**Pull request merged**